opensearch-service-domain-cdk/cdk.context.json

```diff
@@ -0,0 +1,12 @@
+{
+  "domainName": "cdk-os-service-domain",
+  "engineVersion": "OS_1.3",
+  "dataNodeType": "r6g.large.search",
+  "dataNodeCount": 1,
+  "dedicatedMasterNodeType": "r6g.large.search",
+  "dedicatedMasterNodeCount": 3,
+  "warmNodeType": "ultrawarm1.medium.search",
+  "warmNodeCount": 2,
+  "coldStorageEnabled": false,
+  "zoneAwarenessAvailabilityZoneCount": 1
+}
```

Comment:
Things like instance-type availability tend to region specific, so we may need to move some or all of these default into code later so we can do something more clever.  Not a problem for now as long as we're explicit in our documentation and expectations that this is tied to a specific region.  Please make it clear which region this is for.

Reply:
Will do 👍